ios/Classes/SwiftSpotifySdkPlugin.swift

```diff
@@ -204,6 +228,23 @@ public class SwiftSpotifySdkPlugin: NSObject, FlutterPlugin {
                     return
             }
             appRemote.userAPI?.removeItemFromLibrary(withURI: uri, callback: defaultPlayAPICallback)
+        case SpotfySdkConstants.methodGetCapabilities:
+            guard let appRemote = appRemote else {
+                result(FlutterError(code: "Connection Error", message: "AppRemote is null", details: nil))
+                return
+            }
+            appRemote.userAPI?.fetchCapabilities(callback: { (capabilitiesResult, error) in
+                guard error == nil else {
+                    result(FlutterError(code: "UserAPI Error", message: error?.localizedDescription, details: nil))
+                    return
+                }
+                guard let userCapabilities = capabilitiesResult as? SPTAppRemoteUserCapabilities else {
+                    result(FlutterError(code: "UserAPI Error", message: error?.localizedDescription, details: nil))
```

Comment:
```suggestion
                    result(FlutterError(code: "getCapabilitiesError", message: error?.localizedDescription, details: nil))
```
To be consistent with Android